packages/lexical-clipboard/src/clipboard.ts

```diff
@@ -588,6 +588,7 @@ export function setLexicalClipboardDataTransfer(
   clipboardData: DataTransfer,
   data: LexicalClipboardData,
 ) {
+  clipboardData.clearData();
```

Comment:
It looks like that method is unfortunately not available in that context, I guess the next best thing is set all of the defaults.

```suggestion
  for (const [k] of clipboardDataFunctions) {
    if (data[k] === undefined) {
      clipboardData.setData(k, '');
    }
  }
```